webapps/docs/changelog.xml

```diff
@@ -113,6 +113,12 @@
         provided error code during error page processing rather than assuming an
         error code of 500. (markt)
       </fix>
+      <add>
+        Update the HTTP parameter handling to align with the changes in the
+        Jakarta Servlet 6.0 API. Invalid parameters and/or exceeding parameter
```

Comment:
Please also add the section which cases this change.

Reply:
The change is only in the Javadoc. I'll note that in the change log